src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -3825,13 +3851,84 @@ export class StudyViewPageStore {
         default: [],
     });
 
+    private readonly sampleIdentifiersFromGenomicProfileFilter = remoteData({
+        await: () => [this.samples, this.initialMolecularProfileSampleCounts],
+        invoke: async () => {
+            const molecularProfileSampleCountSet = _.keyBy(
+                this.initialMolecularProfileSampleCounts.result || [],
```

Comment:
is there a reason initialMolecularProfileSampleCounts.result would falsy?

Reply:
no